data/map planets.txt

```diff
@@ -748,6 +749,7 @@ planet Darkmetal
 planet Darkrest
 	attributes hai station uninhabited
 	landscape land/station34
+	music ambient/machinery
 	description `This station was mothballed sometime in the last sixty thousand years. The docking area has an atmosphere, and a status panel indicates that the station's generators are functional but shut down, with standby mechanisms running on solar power alone.`
```

Comment:
Maybe we should just leave sounds on these stations until they have proper descriptions?